specs/Workers.md

```diff
@@ -0,0 +1,1053 @@
+[uuid(9f6615b0-08f1-5baa-9c95-a02a1dc56d3f), object, pointer_default(unique)]
+interface ICoreWebView2SharedWorkerCreatedEventArgs : IUnknown {
+  /// The shared worker that was created.
+  [propget] HRESULT Worker([out, retval] ICoreWebView2SharedWorker** value);
+}
+
+/// Receives the result of the `GetSharedWorkers` method.
+[uuid(1f3179ae-15e5-51e4-8583-be0caf85adc7), object, pointer_default(unique)]
+interface ICoreWebView2GetSharedWorkersCompletedHandler : IUnknown {
+  /// Provides the result of the corresponding asynchronous method.
+  HRESULT Invoke([in] HRESULT errorCode, [in] ICoreWebView2SharedWorkerCollectionView* result);
+}
+
+/// Receives `SharedWorkerCreated` events.
+[uuid(79cb8524-b842-551a-8d31-5f824b6955ed), object, pointer_default(unique)]
+interface ICoreWebView2SharedWorkerCreatedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2SharedWorkerManager* sender,
+      [in] ICoreWebView2SharedWorkerCreatedEventArgs* args);
+}
+
+/// A collection of ICoreWebView2SharedWorker.
+[uuid(f8842b09-0108-5575-a965-3d76fd267050), object, pointer_default(unique)]
+interface ICoreWebView2SharedWorkerCollectionView : IUnknown {
+  /// The number of elements contained in the collection.
+  [propget] HRESULT Count([out, retval] UINT32* value);
+
+  /// Gets the element at the given index.
+  HRESULT GetValueAtIndex([in] UINT32 index, [out, retval] ICoreWebView2SharedWorker** value);
+}
+
+[uuid(06cfc21b-56e5-59b1-b7b4-13f197d4539d), object, pointer_default(unique)]
+interface ICoreWebView2Worker : IUnknown {
+  /// A string representing the URL of the script that the worker is executing.
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`.  See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings).
+  [propget] HRESULT ScriptUrl([out, retval] LPWSTR* value);
+
+  /// Add an event handler for the `Destroyed` event that is raised when the worker object is destroyed.
+  /// A worker object is destroyed when the worker script is terminated or when the `CoreWebView2Worker` object is destroyed.
+  HRESULT add_Destroyed(
+      [in] ICoreWebView2WorkerDestroyedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_Destroyed`.
+  HRESULT remove_Destroyed(
+      [in] EventRegistrationToken token);
+}
+
+/// Receives `Destroyed` events.
+[uuid(3197df3b-4245-5fff-9427-e98118fcf657), object, pointer_default(unique)]
+interface ICoreWebView2WorkerDestroyedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2Worker* sender,
+      [in] IUnknown* args);
+}
+
+[uuid(f233edb4-d9b4-5209-9abd-b7f50089464c), object, pointer_default(unique)]
+interface ICoreWebView2DedicatedWorker : IUnknown {
+  /// A string specified when creating a dedicated worker.
```

Comment:
For all the properties, events, methods that correspond to DOM things please say in the ref docs what they correspond to, and how, and link to MDN. Like this one I assume corresponds to the options.name property passed in to Worker ctor?